dss/storage/__init__.py

```diff
@@ -0,0 +1,8 @@
+from typing import NamedTuple, Optional
+
+
+class Tombstone(NamedTuple):
+    """Tombstone object compliant with RFC #4 (Deletion of data in the DCP)"""
```

Comment:
can you provide a permalink or mention the org/repo where this rfc can be found?